### PR TITLE
Add use ipvs to kubeadm-init doc

### PR DESCRIPTION
**Pull request by riverzhang**
how to use ipvs mode with kubeadm

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -260,7 +260,51 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+
+### Using ipvs mode for kube-proxy
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: make this is a sub section of the above. by using `####`

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 2]
```diff
@@ -260,7 +260,51 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: remove empty line.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 3]
```diff
@@ -260,7 +260,51 @@ unifiedControlPlaneImage: ""
...
+### Using ipvs mode for kube-proxy
+
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: please add link to this page:
https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md

and remove the rest of additions in this PR.

we need the two documents to compliment each other.
this introduces duplicate information as per the discussion here:
https://github.com/kubernetes/kubernetes/pull/68075

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 1 above)
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: `ipvs` should be `IPVS`

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 3 above)
Comment (riverzhang) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: ok Thanks

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 1 above)
Comment (riverzhang) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: ok Thanks.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 4]
```diff
@@ -259,8 +259,51 @@ unifiedControlPlaneImage: ""
 
 ### Adding kube-proxy parameters {#kube-proxy}
 
-For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+For information about kube-proxy parameters in the MasterConfiguration see:- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: please change to:
```
For information about kube-proxy parameters in the kubadm configuration see:
- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
```

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 5]
```diff
@@ -259,8 +259,51 @@ unifiedControlPlaneImage: ""
...
+### Using [IPVS](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md) mode for kube-proxy
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: just remove this whole section and add this instead:

```
For information about enabling IPVS mode with kubeadm see:
- [IPVS](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md) mode for kube-proxy
```

**Comment by jbeda** [comment]
/unassign jbeda

Looks like @neolit123 has it.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 2 above)
Comment (neolit123) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: ^ please remove the line. :)

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 6]
```diff
@@ -260,7 +260,52 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+
+For information about enabling IPVS mode with kubeadm see:
+- [IPVS](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md)
+
+In Kubernetes 1.11 and later, Kube-proxy will run in ipvs mode by default in a cluster deployed by kubeadm.
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: please remove the details from lines `267` to `308`
as mentioned on slack this is not needed as this document covers it:
https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md

Comment (riverzhang) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: ok, done.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md (hunk 2 above)
Comment (riverzhang) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: done

**Comment by riverzhang** [comment]
image

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 7]
```diff
@@ -260,7 +260,9 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+For information about enabling IPVS mode with kubeadm see:
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: @riverzhang try adding an empty line above this.

Comment (riverzhang) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: Thanks，it is ok when add empty line.

**Comment by neolit123** [comment]
the preview looks good to me:
https://deploy-preview-10162--kubernetes-io-master-staging.netlify.com/docs/reference/setup-tools/kubeadm/kubeadm-init/

thanks @riverzhang 

/lgtm

**Comment by jbeda** [comment]
@Bradamant3 -- can you approve as necessary?  @riverzhang reached out to me on slack asking me to approve.

**Comment by neolit123** [comment]
/hold remove
@Bradamant3 can we please get an approve here?

**Comment by ryanmcginnis** [comment]
No reason. Didn't know there was a bot that merged automatically. I don't regularly contribute to open-source docs. Now I know. :)

**Comment by mistyhacks** [comment]
It's best to let the bot do it because it runs all the tests (this can take some time). When a PR is both approved and lgtm and not on hold, and has no conflicts (😅) it should merge in a few minutes.

**Pull request merged**